frontend/server/src/Controllers/Problem.php

```diff
@@ -4745,6 +4742,17 @@ public static function getProblemDetailsForTypeScript(
             identityId: intval($r->identity->identity_id)
         );
 
+        $allowedSolutionsTosee = 0;
```

Comment:
Nit:
```suggestion
        $allowedSolutionsToSee = 0;
```

sólo por convención.